packages/styles/scss/components/link-with-icon/_link-with-icon.scss

```diff
@@ -49,23 +49,8 @@
       color: $text-disabled;
     }
 
-    &.#{$prefix}--link-with-icon__icon-left,
```

Comment:
I think this needs to stay in. Removing this breaks the `icon placement` knob in [Storybook](https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/11218/index.html?path=/story/components-link-with-icon--default)